src/applications/personalization/dashboard/components/benefit-payments-and-debts/BenefitPaymentsAndDebt.jsx

```diff
@@ -83,27 +89,33 @@ const BenefitPaymentsAndDebt = ({
               icon="dollar-sign"
               newTab
               text="Manage your direct deposit"
+              /* eslint-disable react/jsx-no-bind */
               onClick={() => {
                 recordEvent({
                   event: 'nav-linkslist',
                   'links-list-header': 'Manage your direct deposit',
                   'links-list-section-header': 'Direct deposit',
                 });
               }}
+              /* eslint-enable react/jsx-no-bind */
+              testId="manage-direct-deposit-link"
             />
             {debtsCount < 1 && (
               <IconCTALink
                 href="/resources/va-debt-management"
                 icon="file-invoice-dollar"
                 newTab
                 text="Learn about VA debt"
+                /* eslint-disable react/jsx-no-bind */

```

Comment:
onClick param does require a callback-function, so I'm disabling the eslint-warning.